pkg/storage/etcd3/store.go

```diff
@@ -58,19 +61,25 @@ type objState struct {
 }
 
 // New returns an etcd3 implementation of storage.Interface.
-func New(c *clientv3.Client, codec runtime.Codec, prefix string) storage.Interface {
-	return newStore(c, codec, prefix)
+func New(c *clientv3.Client, quorumRead bool, codec runtime.Codec, prefix string) storage.Interface {
```

Comment:
@xiang90 - I added the "NewWithNoQuorumRead" func - I think it makes perfect sense.

However, I completely don't understand your argument about "recompile" - this is being set using this flag:
https://github.com/kubernetes/kubernetes/blob/master/pkg/genericapiserver/options/etcd_options.go#L49
and doesn't require recompile.
I think we shouldn't introduce another flag for the same thing (I mean quorum read from etcd).